sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/AccessPointAdapterTest.java

```diff
@@ -92,7 +92,7 @@ void checkFCDALimitations_should_fail_with_one_error_messages() throws Exception
         //Then
         assertThat(sclReportItems).hasSize(1)
                 .extracting(SclReportItem::getMessage)
-                .containsExactlyInAnyOrder("There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST21 in IED IED_NAME");
+                .containsExactlyInAnyOrder("There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST21 in IED IED_NAME : 3 > (max) 2");
```

Comment:
Update issue or roolback changes

Reply:
I updated the issue in Jira and Github for all error messages.